.changeset/purple-pens-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+Increases flow coverage for widget to 100%
```

Comment:
This is going into the changelog for the whole library; I don't think this really tells anyone what's changing since "widget" is pretty generic.
```suggestion
Increases flow coverage for Radio's Widget component to 100%
```

Reply:
Thank you! I went back and forth on whether to add that bit before. Thank you for pointing this out.